Amazon.QLDB.Driver.IntegrationTests/utils/IntegrationTestBase.cs

```diff
@@ -83,6 +85,7 @@ public void RunCreateLedger()
 
         public QldbDriver CreateDriver(
             AmazonQLDBSessionConfig amazonQldbSessionConfig,
+            ISerializer serializer,
```

Comment:
Why not just default this to `null` rather than changing all the callers?

Reply:
Updated to `default` and reverted changing the callers